src/main/java/it/gov/pagopa/controller/consent/PortalConsentControllerImpl.java

```diff
@@ -1,21 +1,17 @@
-package it.gov.pagopa.controller;
+package it.gov.pagopa.controller.consent;
 
 import com.fasterxml.jackson.core.JsonProcessingException;
 import it.gov.pagopa.dto.PortalConsentDTO;
 import it.gov.pagopa.service.PortalConsentService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
-import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.RequestHeader;
-import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
-import java.util.*;
 
 
 @RestController
-//@RequestMapping("/idpay/portal/consent")
 public class PortalConsentControllerImpl implements PortalConsentController {
+
     @Autowired
     PortalConsentService portalConsentService;
```

Comment:
I prefer constructor based injection, but for Controller it could be ok (a Test for a Controller is preferable to be a Spring test, where the @Autowired injections is available)

```suggestion
    private PortalConsentService portalConsentService;
```